### PR TITLE
feature/1 - create form

### PR DESCRIPTION
**Pull request by magnifiq**

### CODE REVIEW

File: form.js [hunk 1]
```diff
@@ -1,26 +1,32 @@
...
+}
 
-  const inputEmail = document.createElement("input");
-  inputEmail.type = "text";
-  inputEmail.placeholder = "Enter your email";
+function createNewAttr(el, atr, val){
+  return el.setAttribute(atr, val);
+}
 
+function appSubEl(parent, child){
```
Comment (igorTT) [inline form.js:12]: Don't reduce append to 'app'.
Get the second parameter as ...rest.
Append children's elements through the cycle.

File: form.js [hunk 2]
```diff
@@ -1,26 +1,32 @@
...
+const form = createNewEl('form');
 
+function createNewEl(el){
+  return document.createElement(el);
+}
 
-  const inputEmail = document.createElement("input");
-  inputEmail.type = "text";
-  inputEmail.placeholder = "Enter your email";
+function createNewAttr(el, atr, val){
```
Comment (igorTT) [inline form.js:8]: It doesn't create something new. It adds an attribute. Get the object as a second parameter.
atr -> attr

File: form.js [hunk 3]
```diff
@@ -1,26 +1,32 @@
-(function () {
-  "use strict";
-  const root = document.getElementById("root");
-  const form = document.createElement("form");
+const root = document.getElementById("root");
+const form = createNewEl('form');
 
+function createNewEl(el){
+  return document.createElement(el);
+}
```
Comment (igorTT) [inline form.js:6]: Let's use function declaration in the places where we need to work with context. Use arrow functions instead to be more consistent.

**Review by igorTT — CHANGES_REQUESTED** [review]
Don't create a separate branch to fix problems from another branch. Return back to the branch where you got the review and fix the problem there instead of creating a new one.

File: form.js [hunk 4]
```diff
@@ -1,26 +1,44 @@
-(function () {
-  "use strict";
-  const root = document.getElementById("root");
-  const form = document.createElement("form");
+const root = document.getElementById("root");
 
+const createNewEl=(el)=>{
```
Comment (igorTT) [inline form.js:3]: should be empty space betwwen function name, etc...
```javascript
const createNewEl = (el) => {
...
```

File: form.js [hunk 5]
```diff
@@ -1,26 +1,44 @@
...
+  return document.createElement(el);
+}
```
Comment (igorTT) [inline form.js:5]: Missing semicolon

**Review by igorTT — CHANGES_REQUESTED** [review]
Configure IDE to fix such problems automaticaly

**Review by igorTT — CHANGES_REQUESTED** [review]
I don't see any changes since the last code review.

**Comment by magnifiq** [comment]
I committed changes to bugfix/3 branch

**Comment by igorTT** [comment]
> I committed changes to bugfix/3 branch

But I added a comment to the current pull request, not to the bugfix/3

**Review by igorTT — APPROVED** [review]

**Pull request merged**